### PR TITLE
dxToolbar - Fix internal menu styles

### PR DESCRIPTION
**Pull request by groshenkovamarina**

### CODE REVIEW

File: styles/widgets/generic/toolbar.generic.less [hunk 1]
```diff
@@ -180,3 +189,21 @@
...
 }
+
+.dx-toolbar-button {
+    .dx-menu {
+        .dx-menu-item {
+            border: 1px solid transparent;
+            border-radius: 4px;
+
+            .dx-menu-item-content {
+                padding: @GENERIC_TOOLBAR_MENU_TOP_PADDING @GENERIC_MENU_ITEM_PADDING_BASE @GENERIC_TOOLBAR_MENU_BOTTOM_PADDING;
```
Comment (babich-a) [inline styles/widgets/generic/toolbar.generic.less]: if horizontal menu padding is not changed anywhere before I suggest using` padding-top` and `padding-bottom`. the using of @GENERIC_MENU_ITEM_PADDING_BASE constant here is not good for future module system.

**Review by babich-a — DISMISSED** [review]

File: styles/widgets/generic/toolbar.generic.less [hunk 2]
```diff
@@ -180,3 +189,22 @@
 .dx-toolbar .dx-tab {
     padding: 4px;
 }
+
+.dx-toolbar-button {
+    .dx-menu {
+        .dx-menu-item {
+            border: 1px solid transparent;
+            border-radius: 4px;
```
Comment (BingoRUS) [inline styles/widgets/generic/toolbar.generic.less]: Should we use @base-border-radius here?

Comment (groshenkovamarina) [reply styles/widgets/generic/toolbar.generic.less]: A menu item in a toolbar should look like a button so I use button's parameters here

**Review by BingoRUS — APPROVED** [review]

File: styles/widgets/generic/toolbar.generic.less (hunk 2 above)
Comment (BingoRUS) [reply styles/widgets/generic/toolbar.generic.less]: @button-border-radius is fine too.

**Pull request merged**